File: articles/cognitive-bias-institutional-decisions/fortran/bias_pressure_score.f90
! Cognitive Bias in Institutional Decisions
! Simple Fortran scoring demonstration.
!
! Responsible-use scope:
! Synthetic-data research, methods demonstration, institutional learning,
! and reproducible workflows only. Not for employment, executive ranking,
! productivity ranking, or individual worker decisions.

program bias_pressure_score_program
  implicit none
  real :: signal_quality, viewpoint_diversity, analytical_discipline
  real :: structured_challenge, hierarchy_suppression, time_pressure
  real :: routine_reinforcement, learning_review, group_conformity
  real :: score

  signal_quality = 84.0
  viewpoint_diversity = 80.0
  analytical_discipline = 82.0
  structured_challenge = 81.0
  hierarchy_suppression = 14.0
  time_pressure = 38.0
  routine_reinforcement = 36.0
  learning_review = 79.0
  group_conformity = 28.0

  score = 0.16 * hierarchy_suppression + 0.15 * time_pressure + &
          0.14 * routine_reinforcement + 0.13 * group_conformity - &
          0.16 * viewpoint_diversity - 0.13 * analytical_discipline - &
          0.14 * structured_challenge - 0.12 * learning_review - &
          0.10 * signal_quality

  print *, "Synthetic bias pressure score:", score
  print *, "Responsible-use reminder: institutional learning only."
end program bias_pressure_score_program

File: articles/cognitive-bias-institutional-decisions/go/bias_pressure_score.go
// Cognitive Bias in Institutional Decisions
// Simple Go scoring demonstration.
//
// Responsible-use scope:
// Synthetic-data research, methods demonstration, institutional learning,
// and reproducible workflows only. Not for employment, executive ranking,
// productivity ranking, or individual worker decisions.

package main

import "fmt"

type BiasPressureProfile struct {
	SignalQuality        float64
	ViewpointDiversity   float64
	AnalyticalDiscipline float64
	StructuredChallenge  float64
	HierarchySuppression float64
	TimePressure         float64
	RoutineReinforcement float64
	LearningReview       float64
	GroupConformity      float64
}

func BiasPressureScore(p BiasPressureProfile) float64 {
	return 0.16*p.HierarchySuppression +
		0.15*p.TimePressure +
		0.14*p.RoutineReinforcement +
		0.13*p.GroupConformity -
		0.16*p.ViewpointDiversity -
		0.13*p.AnalyticalDiscipline -
		0.14*p.StructuredChallenge -
		0.12*p.LearningReview -
		0.10*p.SignalQuality
}

func main() {
	profile := BiasPressureProfile{84, 80, 82, 81, 14, 38, 36, 79, 28}
	fmt.Printf("Synthetic bias pressure score: %.2f\n", BiasPressureScore(profile))
	fmt.Println("Responsible-use reminder: institutional learning only.")
}
